packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/SelectedPeopleList.tsx

```diff
@@ -1,19 +1,52 @@
 /* tslint:disable */
 import * as React from 'react';
 /* tslint:enable */
-import { SelectedItemsList } from '../SelectedItemsList';
 import { SelectedPersona } from './Items/SelectedPersona';
-import { ISelectedItemsListProps, ISelectedItemsList, BaseSelectedItem } from '../SelectedItemsList.types';
+import {
+  BaseSelectedItem,
+  IControlledSelectedItemListProps,
+  IUncontrolledSelectedItemListProps,
+  IControlledSelectedItemsList,
+  IUncontrolledSelectedItemsList
+} from '../SelectedItemsList.types';
 import { IPersonaProps } from 'office-ui-fabric-react/lib/Persona';
+import { ControlledSelectedItemsList } from '../ControlledSelectedItemsList';
+import { UncontrolledSelectedItemsList } from '../UncontrolledSelectedItemsList';
+import { MaybeControlled } from '../MaybeControlled';
 
 type PartiallyOptional<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>> & Pick<Partial<T>, keyof T>;
-export type ISelectedPeopleListProps<TPersona> = PartiallyOptional<ISelectedItemsListProps<TPersona>, 'onRenderItem'>;
+type PartiallyOptionalWithoutOnRenderItem<T extends { onRenderItem: any }> = PartiallyOptional<T, 'onRenderItem'>;
 
-export type ISelectedPeopleList<TPersona extends IPersonaProps & BaseSelectedItem = IPersonaProps> = ISelectedItemsList<TPersona>;
-export type SelectedPeopleList<TPersona extends IPersonaProps & BaseSelectedItem = IPersonaProps> = SelectedItemsList<TPersona>;
-export const SelectedPeopleList = React.forwardRef(
-  <TPersona extends IPersonaProps & BaseSelectedItem = IPersonaProps>(
-    props: ISelectedPeopleListProps<TPersona>,
-    ref: React.Ref<ISelectedPeopleList<TPersona>>
-  ) => <SelectedItemsList<TPersona> ref={ref} onRenderItem={SelectedPersona} {...props} />
-);
+export interface IControlledSelectedPeopleListProps<TPersona>
+  extends PartiallyOptionalWithoutOnRenderItem<IControlledSelectedItemListProps<TPersona>> {}
+export interface IUncontrolledSelectedPeopleListProps<TPersona>
```

Comment:
Nit I think some spaces in here might make this more readable.

Reply:
This is just run through `prettier`, I think?